src/App.tsx

```diff
@@ -1,41 +1,63 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+import { getAllUsers } from './api/api';
+import { User } from './react-app-env';
```

Comment:
it would be a great benefit if you [order your imports somehow](https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf)